var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -104,6 +100,8 @@ class CbtfKrell(Package):
 
```

Comment:
We can do spack install cbtf-krell +openmpi +mvapich2 +mpich to create multiple mpi performance data collectors.